cmake/testing.cmake

```diff
@@ -233,11 +233,11 @@ if(VTK_VERSION VERSION_GREATER_EQUAL 9.1.20220117)
   f3d_test(NAME TestHDRI HDRI DATA suzanne.ply ARGS --hdri=${CMAKE_SOURCE_DIR}/testing/data/palermo_park_1k.hdr DEFAULT_LIGHTS)
   f3d_test(NAME TestHDRIBlur HDRI DATA suzanne.ply ARGS -u --hdri=${CMAKE_SOURCE_DIR}/testing/data/palermo_park_1k.hdr DEFAULT_LIGHTS)
   f3d_test(NAME TestHDRIBlurRatio HDRI DATA suzanne.ply RESOLUTION 600,100 ARGS -u --hdri=${CMAKE_SOURCE_DIR}/testing/data/palermo_park_1k.hdr DEFAULT_LIGHTS)
-  f3d_test(NAME TestHDRIEdges HDRI DATA suzanne.ply ARGS -e --hdri=${CMAKE_SOURCE_DIR}/testing/data/palermo_park_1k.hdr DEFAULT_LIGHTS)
-  f3d_test(NAME TestHDRI8Bit DATA suzanne.ply ARGS --hdri=${CMAKE_SOURCE_DIR}/testing/data/logo.tif HDRI)
-  f3d_test(NAME TestHDRIOrient DATA suzanne.stl ARGS --up=+Z --hdri=${CMAKE_SOURCE_DIR}/testing/data/palermo_park_1k.hdr HDRI)
+  f3d_test(NAME TestHDRIEdges HDRI DATA suzanne.ply THRESHOLD 400 ARGS -e --hdri=${CMAKE_SOURCE_DIR}/testing/data/palermo_park_1k.hdr DEFAULT_LIGHTS)
+  f3d_test(NAME TestHDRI8Bit DATA suzanne.ply THRESHOLD 100 ARGS --hdri=${CMAKE_SOURCE_DIR}/testing/data/logo.tif HDRI)
+  f3d_test(NAME TestHDRIOrient DATA suzanne.stl THRESHOLD 400 ARGS --up=+Z --hdri=${CMAKE_SOURCE_DIR}/testing/data/palermo_park_1k.hdr HDRI)
```

Comment:
what about this one ?

Reply:
Baseline:
![TestHDRIOrient](https://user-images.githubusercontent.com/6069752/211142036-261f248c-ebc6-4710-8220-4376b41d79c7.png)

Computed:
![TestHDRIOrient](https://user-images.githubusercontent.com/6069752/211142044-bc570ccb-38ca-4b0b-badd-54e650ce06a0.png)

I suspect it's because of precision changes of PBR textures, but I can't really change the baselines without breaking tests for older VTK versions.

Reply:
Disabling HDRi test for older VTK version would be better imo.